beets/library.py

```diff
@@ -451,6 +451,9 @@ class Item(LibModel):
         'lyricist':             types.STRING,
         'composer':             types.STRING,
         'composer_sort':        types.STRING,
+        'work':                 types.STRING,
+        'work_id':              types.STRING,
```

Comment:
Other MusicBrainz IDs (which are listed lower down in this file) are called `mb_trackid`, `mb_albumid`, etc. Can we please match that naming pattern?

Reply:
done